Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -922,6 +1334,35 @@ def incident_attachments(client, incident_id):
     return response
 
 
+def upload_incident_attachment(client: SimpleClient, incident_id: str, entry_id: str, tag_to_ibm: str):
+    """
+        Uploads a file from XSOAR to the IBM QRadar SOAR incident with ID `incident_id`.
+    """
+    try:
+        file_path_obj = demisto.getFilePath(entry_id)
+    except ValueError:
+        return CommandResults(
+            entry_type=EntryType.ERROR,
+            readable_output=f"Could not find a file with entry ID: {entry_id}",
+        )
+
+    file_path, file_name = file_path_obj.get("path"), file_path_obj.get("name")
+    if '.' in file_name:
```

Comment:
consider using os.path.splitext and then check if it has file extension or not.

Reply:
adapted the code accordingly with os.path.splitext